test/com/facebook/buck/swift/testdata/swift_test_with_host/BUCK.fixture

```diff
@@ -0,0 +1,49 @@
+swift_library(
```

Comment:
I think we should be testing `apple_library` in the very least, having a separate test for `swift_library` might be beneficial, too.

Reply:
Agreed, I'll change it to test both.